pages/careers/_open-positions.html

```diff
@@ -261,7 +265,100 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
       >
     </p>
 
-    <hr />
+    <hr /> 
+
+    <h3>Software Engineering (Contract)</h3>
```

Comment:
```suggestion
    <h3 id=software-engineering-contractor>Software Engineering (Contract)</h3>
```
I think this might make the href work